src/Nethermind/Nethermind.Consensus/Processing/BlockProcessor.cs

```diff
@@ -310,13 +310,13 @@ protected virtual TxReceipt[] ProcessBlock(
         ReceiptsTracer.SetOtherTracer(blockTracer);
         ReceiptsTracer.StartNewBlockTrace(block);
 
-        StoreBeaconRoot(block, spec);
+        var blkCtx = new BlockExecutionContext(block.Header, spec);
+
+        StoreBeaconRoot(block, in blkCtx, spec);
```

Comment:
shouldn't here context be with 0 base fee?

Reply:
Is using same context; was using same header and spec to create `new BlockExecutionContext(header, spec)`; so changed to pass it though